cmd/main.go

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright 2024 Canonical, Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"errors"
+	"net/http"
+	"os"
+
+	log "github.com/sirupsen/logrus"
```

Comment:
nit: please use the standard log library. `logrus` is in maintenance mode. https://github.com/sirupsen/logrus

I know that we have some left-over usage in our codebase but for new projects we should not use it anymore.

Reply:
Done.